### PR TITLE
Add docs about mgr-sign-metadata-ctl

### PR DESCRIPTION
**Pull request by Loquacity**
From https://github.com/SUSE/doc-susemanager/pull/355

@mateiw

### CODE REVIEW

File: modules/administration/pages/repo-metadata.adoc [hunk 1]
```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
...
-=== Generate a Custom GPG key
+You will require a custom GPG key to be able to sign repository metadata.
 
-To sign repository metadata a custom GPG key is required. Create a new GPG key as *root* via the following steps.
+.Procedure: Generating a custom GPG Key
 
+. As the root user, use the [command]``gpg`` command to generate a new key:
++
 ----
 $> gpg --gen-key
```
Comment (keichwa) [inline modules/administration/pages/repo-metadata.adoc]: I'd vote to get rid of the command-line indicator (`$>`) if this example does not print the command output.

File: modules/administration/pages/repo-metadata.adoc [hunk 2]
```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
...
 +
+----
+mgr-sign-metadata-ctl regen-metadata
+----
+
+You can also use the [command]``mgr-sign-metadata-ctl`` command to perform other tasks.
+Use [command]``mgr-sign-metadata-ctl --help`` to see the complete list.
+
+Repository metadata signing is a global option.
+Once it is enabled, it is enabled on all software channels on the server.
```
Comment (keichwa) [inline modules/administration/pages/repo-metadata.adoc]: "When" per style guide

File: modules/administration/pages/repo-metadata.adoc [hunk 3]
```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
...
-.Tip
-====
-For salt managed systems it might make sense to use a state to trust GPG keys.
-====
+.Procedure: Importing GPG keys on Clients
+
+. On RPM-based client systems, use these remote commands:
++
+----
+rpm --import http://<server.domain.top/pub/keyname.key
```
Comment (keichwa) [inline modules/administration/pages/repo-metadata.adoc]: Better use: `rpm --import http://server.example.org/pub/keyname.key` (remove the stray `<` and use "example" address).  For the next command, you can use `example.com`.

Comment (Loquacity) [reply modules/administration/pages/repo-metadata.adoc]: hrm, not sure if the 'server' is required? What do you think?

Comment (keichwa) [reply modules/administration/pages/repo-metadata.adoc]: For clarity, I'd keep the "server" part.  I'm not that sure whether we want to have two different domain names or not.  Do we actually need two import statements, @mateiw ?

Also wondering now why the rpm import commands are labeled as _remote_ commands.

File: modules/administration/pages/repo-metadata.adoc [hunk 4]
```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
...
++
 ----
-# gpg --batch --export -a -o <output filename> <KEYID>
-$> gpg --batch --export -a -o /srv/www/htdocs/pub/company.key 607FABDB
+mgr-sign-metadata-ctl enable 4E2C3DD8
+OK. Found key 4E2C3DD8 in keyring.
+DONE. Set key 4E2C3DD8 in /etc/rhn/signing.conf.
+DONE. Enabled metadata signing in /etc/rhn/rhn.conf.
+DONE. Exported key 4E2C3DD8 to /srv/susemanager/salt/gpg/mgr-keyring.gpg.
+DONE. Exported key 4E2C3DD8 to /srv/www/htdocs/pub/mgr-gpg-pub.key.
```
Comment (mateiw) [inline modules/administration/pages/repo-metadata.adoc]: Missing lines at the end of the output: 
```
NOTE. For the changes to become effective run:
   mgr-sign-metadata-ctl regen-metadata

```

File: modules/administration/pages/repo-metadata.adoc [hunk 5]
```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
...
-====
+.Procedure: Importing GPG keys on Clients
+
+. On RPM-based client systems, use these remote commands:
++
+----
+rpm --import http://server.example.com/pub/keyname.key
+rpm --import http://server.example.com/pub/company.key
+----
+. For Ubuntu clients, you will need to reassign the channels, which will automatically pick up the new GPG key.
```
Comment (mateiw) [inline modules/administration/pages/repo-metadata.adoc]: Reassigning the channels can be done either from the UI (the preferred method) or from the server command line: `salt <ubuntu-minion> state.apply channels`.

**Pull request merged**